packages/blockchain-link/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-# 2.1.4 (not released)
+# 2.1.4
 
 -   add missing ADA types
+-   added `join` transaction type to Transaction interface. removed `totalSpent` field from Transaction interface
```

Comment:
It's called <code>join<strong>t</strong></code> transaction.

Maybe we could also mention that there is new `isAccountOwned` flag in `tx.details.vin`/`tx.details.vout` items (used for highlighting user inputs & outputs).